roles/forklift_versions/molecule/default/verify.yml

```diff
@@ -74,7 +60,7 @@
       vars:
         scenario: "{{ pipeline_type }}"
         scenario_os: "{{ pipeline_os }}"
-        scenario_version: "4.9"
+        scenario_version: "4.8"
```

Comment:
This test does upgrade testing -- but it jumps two versions.
I guess the easiest for that is to keep version at 4.9 here, but set `scenario_os` to `almalinux8` explicitly, then it'll pass again.

(We do not have enough el9-based versions to test this otherwise)